### PR TITLE
add Address entity

### PR DESCRIPTION
**Pull request by Rodrick2550**
Hi fernando, this is entitie address, if there is any mistake please let me know asap, have a nice night

### CODE REVIEW

**Comment by devrrior** [comment]
The attributes don't need to be unique

File: src/main/java/com/school/ecommerceupch/entities/Address.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@Entity
+@Table(name = "addresses")
+@Getter @Setter
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    @Column(unique = true, length = 255)
```
Comment (devrrior) [inline src/main/java/com/school/ecommerceupch/entities/Address.java]: It's not necessary the unique

**Comment by Rodrick2550** [comment]
Hi fernando, i made the changes that you write me, please if there's anything else that i have to change please let me know asap

**Review by devrrior — COMMENTED** [review]
good!

**Pull request merged**